chart/env/prod.yaml

```diff
@@ -183,12 +183,12 @@ api:
 
 workers:
   -
-    deployName: "generic"
+    deployName: "all"
     maxJobsPerNamespace: 20
-    workerOnlyJobTypes: ""
+    workerOnlyJobTypes: "/config-names,/split-names-from-streaming,splits,/first-rows,/parquet-and-dataset-info,"
```

Comment:
same comment, I would let it empty

Reply:
Done in https://github.com/huggingface/datasets-server/pull/945/commits/47e16e3cdb88b1005c34bec34c181162008fe1d3